src/graphics-path.c

```diff
@@ -1236,14 +1234,28 @@ GdipAddPathString (GpPath *path, GDIPCONST WCHAR *string, int length,
 	GpRectF box;
 	GpPointF box_offset;
 	PangoLayout* layout; 
+	GpStringFormat *string_format = (GpStringFormat *) format;
 
-	cairo_save (cr);
-	layout = gdip_pango_setup_layout ((GpGraphics *)cr, string, length, font, layoutRect, &box, &box_offset, format, NULL);
+	if (!(format->formatFlags & StringFormatFlagsNoClip)) {
```

Comment:
The format is optional so this will crash with a null pointer dereference if the format is not passed

Reply:
Right... will fix.